### PR TITLE
SPARK-27509 Introduce Helm

### PR DESCRIPTION
**Pull request by djasiok**

### CODE REVIEW

**Comment by djasiok** [comment]
run build

**Comment by djasiok** [comment]
run build

**Comment by djasiok** [comment]
run build

File: helm/appstore-metadata-service/.helmignore [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#  If not stated otherwise in this file or this component's LICENSE file the
+#  following copyright and licenses apply
+#
+#  Copyright 2020 Liberty Global B.V.
```
Comment (patrox) [inline helm/appstore-metadata-service/.helmignore:4]: It's 2021 already

File: helm/appstore-metadata-service/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      paths:
+      - path: /
+        backend:
+          serviceName: {{ include "appstore-metadata-charts.fullname" . }}
+          servicePort: 8080
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "appstore-metadata-charts.fullname" . }}-as3
```
Comment (patrox) [inline helm/appstore-metadata-service/templates/ingress.yaml:65]: `as3` This is a bit meaningless as references a non-existing service.
In the past this was called `maintainers` context - as it was referencing something which is much clearer,
as it says clearly who can access this part.

**Review by patrox — APPROVED** [review]

File: helm/appstore-metadata-service/.helmignore (hunk 1 above)
Comment (djasiok) [reply helm/appstore-metadata-service/.helmignore]: done

File: helm/appstore-metadata-service/templates/ingress.yaml (hunk 2 above)
Comment (djasiok) [reply helm/appstore-metadata-service/templates/ingress.yaml:65]: after discussing with the Component Owner we decided to leave it as it is

**Comment by djasiok** [comment]
run build

**Review by AnDyXX — APPROVED** [review]

**Review by karolostaszewski — APPROVED** [review]

**Pull request merged**